src/main/java/com/google/sps/data/PollingStation.java

```diff
@@ -16,6 +16,9 @@
 
 import com.google.appengine.api.datastore.Entity;
 import com.google.auto.value.AutoValue;
+import com.google.common.collect.ImmutableList;
+import java.util.ArrayList;
```

Comment:
I think this is unused now?

Reply:
I still cast to an ArrayList when setting the sources property for the PollingStation entity in Datastore (line 221).